### DISCUSSION TITLE
how do i disable both public pages and radio.mp3 file?

### DISCUSSION THREAD

**Discussion by dolphin-cat** [AzuraCast/AzuraCast#5074] [Q&A]
i need to make a paid radio service with wordpress and i am using azuracast to play the music, problem is everyone can go to public pages and listen for free and if i disable public pages, everyone can still go to radio.mp3 file and listen for free

**Comment by LordFader** [comment]
Edit Station>Public Pages>Enable/Disable... (@ the bottom)
The "radio.mp3"  ...??? 
Are you talking about the Azuracast Message?
Edit Station>Mount Point>Basic Info>Fallback/error.mp3
Remove that "/error.mp3" or change for a fallback you want/need
